mercury/views/simulator.py

```diff
@@ -13,40 +25,91 @@ class SimulatorView(TemplateView):
-            wheel_speed_fl=post_wheel_speed_fl,
-            wheel_speed_br=post_wheel_speed_br,
-            wheel_speed_bl=post_wheel_speed_bl,
-            suspension_fr=post_suspension_fr,
-            suspension_fl=post_suspension_fl,
-            suspension_br=post_suspension_br,
-            suspension_bl=post_suspension_bl,
-            current_fuel_level=post_current_fuel_level,
-        )
-        sim_data.save()
+
+        if request.POST.get("created_at_accel"):
+            post_created_at_accel = request.POST.get("created_at_accel")
+            post_acceleration_x = request.POST.get("acceleration_x")
+            post_acceleration_y = request.POST.get("acceleration_y")
+            post_acceleration_z = request.POST.get("acceleration_z")
+
+            accel_data = AccelerationSensor(
+                created_at_accel=post_created_at_accel,
+                acceleration_x=post_acceleration_x,
+                acceleration_y=post_acceleration_y,
+                acceleration_z=post_acceleration_z,
+            )
+            accel_data.save()
+
+        elif request.POST.get("created_at_ws"):
+
+            post_created_at_ws = request.POST.get("created_at_ws")
+            post_wheel_speed_fr = request.POST.get("wheel_speed_fr")
+            post_wheel_speed_fl = request.POST.get("wheel_speed_fl")
+            post_wheel_speed_br = request.POST.get("wheel_speed_br")
+            post_wheel_speed_bl = request.POST.get("wheel_speed_bl")
+
+            ws_data = WheelSpeedSensor(
+                created_at_ws=post_created_at_ws,
+                wheel_speed_fr=post_wheel_speed_fr,
+                wheel_speed_fl=post_wheel_speed_fl,
+                wheel_speed_br=post_wheel_speed_br,
+                wheel_speed_bl=post_wheel_speed_bl,
+            )
+            ws_data.save()
+
+        elif request.POST.get("created_at_ss"):
+            post_created_at_ss = request.POST.get("created_at_ss")
+            post_suspension_fr = request.POST.get("suspension_fr")
+            post_suspension_fl = request.POST.get("suspension_fl")
+            post_suspension_br = request.POST.get("suspension_br")
+            post_suspension_bl = request.POST.get("suspension_bl")
+
+            ss_data = SuspensionSensor(
+                created_at_ss=post_created_at_ss,
+                suspension_fr=post_suspension_fr,
+                suspension_fl=post_suspension_fl,
+                suspension_br=post_suspension_br,
+                suspension_bl=post_suspension_bl,
+            )
+            ss_data.save()
+
+        elif request.POST.get("created_at_fl"):
+            post_created_at_fl = request.POST.get("created_at_fl")
+            post_current_fuel_level = request.POST.get("current_fuel_level")
+
+            fl_data = FuelLevelSensor(
+                created_at_fl=post_created_at_fl,
+                current_fuel_level=post_current_fuel_level,
+            )
+            fl_data.save()
+
+        else:
```

Comment:
I think you should still check if the data submitted is for the temperature sensor instead of stuffing this in the last-case else block.